web/client/plugins/Identify.jsx

```diff
@@ -53,7 +53,8 @@ const selector = createStructuredSelector({
     dockStyle: state => mapLayoutValuesSelector(state, {height: true}),
     formatCoord: (state) => state.mapInfo && state.mapInfo.formatCoord,
     showCoordinateEditor: (state) => state.mapInfo && state.mapInfo.showCoordinateEditor,
-    showEmptyMessageGFI: state => showEmptyMessageGFISelector(state)
+    showEmptyMessageGFI: state => showEmptyMessageGFISelector(state),
+    isCesium
```

Comment:
Related to previous comment on `enable` prop, is it possible to merge `isCesium` in enable ?